apps/src/lib/kits/maker/boards/microBit/MicroBitConstants.js

```diff
@@ -53,3 +53,40 @@ export const MB_COMPONENT_EVENTS = {
 MB_SENSOR_VARS.forEach(sensor => (MB_COMPONENT_EVENTS[sensor] = SENSOR_EVENTS));
 
 export const MICROBIT_FIRMWARE_VERSION = 'micro:bit Firmata 1.1';
+
+export const SQUARE_LEDS = [
```

Comment:
great call to define these LED matrix constants here to make the other code more readable!